include/deal.II/trilinos/rol_vector.h

```diff
@@ -133,69 +140,117 @@ namespace TrilinosWrappers
      */
     ROL::Ptr<VectorType> vector_ptr;
 
+    /**
+     * IndexSet that represents the locally owned optimization space.
+     */
+    IndexSet optimization_space;
+
+    /**
+     * Global dimension of the optimization space.
+     */
+    IndexSet::size_type global_dimension;
+
+    /**
+     * Prefix sum of the number of elements in the optimization space of lower
+     * rank MPI processes.
+     */
+    IndexSet::size_type prefix_sum;
```

Comment:
Please give this variable a better name that says more than just "sum" (of what?). Perhaps `local_opt_var_starting_index`? I imagine you can come up with a better name than me.